frontend/src/compositions/useNotifications.ts

```diff
@@ -28,6 +28,14 @@ export default function useNotifications() {
       life: 3000,
     });
   };
+  const showCreateArticleNotification = () => {
+    toast.add({
+      severity: "info",
+      summary: "Info",
+      detail: "Ein neuer Artikel wurde erstellt",
```

Comment:
Just out of curiosity: Do we need the summary? Maybe this is enough (for all toasts)

```suggestion
      summary: "Ein neuer Artikel wurde erstellt",
```